src/rgw/motr/gc/gc.cc

```diff
@@ -14,16 +14,60 @@
  */
 
 #include "motr/gc/gc.h"
+#include <cstdlib>
+#include <ctime>
 
 void *MotrGC::GCWorker::entry() {
   std::unique_lock<std::mutex> lk(lock);
   ldpp_dout(dpp, 10) << __func__ << ": " << gc_thread_prefix
     << worker_id << " started." << dendl;
 
+  // Get random number to lock the GC index.
+  uint32_t my_index = rand() % (motr_gc->max_indices);
+  // This is going to be endless loop
   do {
 
     ldpp_dout(dpp, 10) << __func__ << ": " << gc_thread_prefix
-      << worker_id << " iteration" << dendl;
+      << worker_id << " Iteration Started" << dendl;
+
+    std::string iname = "";
+    // Get lock on an GC index
+    int rc = motr_gc->get_locked_gc_index (&my_index);
+
+    // Lock has been aquired, start the timer
+    std::time_t start_time = std::time(nullptr);
+    std::time_t end_time = start_time + \
+			   cct->_conf->rgw_gc_processor_max_time - 10;
+    std::time_t current_time = std::time(nullptr);
+    uint32_t processed_count = 0;
```

Comment:
Please declare `processed_count` inside if statement below to resolve codacy warning.

Reply:
Done